### PR TITLE
deployment: fix trailing whitespace errors.

### PR DESCRIPTION
**Pull request by klihub**
- fix trailing WS errors that slipped through in Helm Chart READMEs

### CODE REVIEW

**Review by fmuyassarov — COMMENTED** [review]
It looks good to me but I have a question. I'm totally new to githooks so it might sound stupid what I ask. What's the benefit of having git hook way of running checks to catch trailing whitespaces over having a workflow step that would run a script for that?

**Comment by klihub** [comment]
> It looks good to me but I have a question. I'm totally new to githooks so it might sound stupid what I ask. What's the benefit of having git hook way of running checks to catch trailing whitespaces over having a workflow step that would run a script for that?

The idea is to provide a low barrier for checking things that are either wrong (eg. accidentally committed conflict markers), or undesired and probably caught by reviewers, therefore preventing the PR from getting merged. OR to put in another way, to avoid (mis)using scarce common infra resources (shared github workers) to perform problems that can be trivially detected locally.

This goes in the same bucket as our `verify-fmt`, `verify-godeps`, 'docs` etc. targets in the `Makefile` which allow you to pre-check locally some of the things our github workflows might be picking on and the idea is that you run them locally before filing a PR. We should probably add a single `pre-pr-checks` target to run all of these, so it'd be easier to remember what to run before a PR.

**Comment by klihub** [comment]
> It looks good to me but I have a question. I'm totally new to githooks so it might sound stupid what I ask. What's the benefit of having git hook way of running checks to catch trailing whitespaces over having a workflow step that would run a script for that?

@fmuyassarov  Sorry for the inconvenience but I split out all but the whitespace fixup commit to #171.

File: .github/workflows/project-checks.yaml [hunk 1]
```diff
@@ -27,6 +27,15 @@ jobs:
...
 
+      - name: Git-check PR for conflict markers and trailing whitespace errors.
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
+          git fetch -q origin ${{ github.base_ref }}
+          if ! git diff-index --check origin/${{ github.base_ref }} --; then
+              echo "This PR would introduce the above errors."
+              echo "Please fix them and update the PR."
+              exit 1
+          fi
```
[the remark this answers is not in the document]
Comment (klihub) [reply .github/workflows/project-checks.yaml]: > I was first to comment out if we really need L31 (if operator) assuming we run checks only on pull requests. But in the next steps below, I see that we are using this workflow to build the site. Maybe later on we need to seperate site building steps from verification/tests against pull requests.

We can do that, but some consideration should be given whether to go with a large number of small workflows, which again tend to waste common shared resources, or larger blocks which can be customized... ideally are decomposed into smaller reusable 'sub-workflows` as demonstrated by how @marquiz rewrote the CRI Resource Manager workflows using `on/workflow_call/inputs` and `uses` tags together with locally maintained common sub-workflows.

**Review by marquiz — APPROVED** [review]
Thx @klihub 
LGTM

**Comment by klihub** [comment]
> Also, it would be good to rename the PR title because it is too long and mix of commit titles I think.

Rewrote the title.

**Review by fmuyassarov — APPROVED** [review]
LGTM

**Pull request merged**